packages/plugin-table/web/src/modals/TableColorPicker.tsx

```diff
@@ -0,0 +1,57 @@
+import type { FC } from 'react';
+import React, { useContext } from 'react';
+import { RicosContext, ToolbarContext } from 'ricos-context';
+import { getLangDir } from 'wix-rich-content-common';
+import { ColorPicker } from 'wix-rich-content-plugin-commons';
+import { DEFAULT_PALETTE_BG } from '../TableToolbar/CellFormattingButtonProps';
+import { colorPicker } from './colorPicker';
+
+type Props = {
+  closeModal: () => void;
+  onChange: (color) => void;
+  resetColor: () => void;
+  palette: string[];
+  onColorAdded?: (color) => void;
+  currentColor?: string;
+};
+
+export const TableColorPicker: FC<Props> = ({
+  closeModal,
+  onChange,
+  resetColor,
+  currentColor,
+  onColorAdded,
+  palette,
+}) => {
+  const { isMobile, theme, locale } = useContext(ToolbarContext);
```

Comment:
I think ToolbarContext will be deprecated
```suggestion
  const { isMobile, theme, locale } = useContext(RicosContext);
```